src/javascript/app/pages/user/set_currency.js

```diff
@@ -109,6 +105,27 @@ const SetCurrency = (() => {
                 } else {
                     $error.text(localize('Please choose a currency')).setVisibility(1);
                 }
+            };
+
+            $('.currency_wrapper').on('click', function () {
+                $currency_list.find('> div').removeClass('selected');
+                $(this).addClass('selected');
```

Comment:
I would suggest storing $(this) in a variable since it's used more than once

Reply:
it would be even better if it could be changed from jquery to javascript

Reply:
done